package.json

```diff
@@ -7,12 +7,14 @@
   "devDependencies": {
```

Comment:
This was updated automatically in the process of installing dependencies and running tests.